server/index.js

```diff
@@ -44,9 +44,33 @@ app.get('/api/check/:gameId', (req, res) => {
   const params = [gameId, userId];
   db.query(sql, params)
     .then(result => {
-      const [todo] = result.rows;
-      if (todo !== undefined) {
-        res.status(201).json(todo);
+      const [game] = result.rows;
+      if (game !== undefined) {
+        res.status(201).json(game);
+      } else {
+        res.status(201).json({ notInDb: 'not in database' });
+      }
+    })
+    .catch(err => {
+      console.error(err);
+      res.status(500).json({
+        error: 'an unexpected error occurred'
+      });
+    });
+});
+
+app.get('/api/favorites', (req, res) => {
+  const userId = 1;
+  const sql = `
+    select *
+    from "favorites"
+    where "userId" = $1
+  `;
+  const params = [userId];
+  db.query(sql, params)
+    .then(result => {
+      if (result.rows !== undefined) {
```

Comment:
A `pg.Result` object always has a `rows` property, even if it's empty (`result.rows.length === 0`).